README.md

```diff
@@ -1,33 +1,41 @@
 # Scalameta language-server
 
-This is an experimental [Language Server Protocol](https://github.com/Microsoft/language-server-protocol)
-implementation for Scala based on Scalameta.
+This project is an experiment to implement a [Language Server](https://github.com/Microsoft/language-server-protocol)
+for Scala using Scalameta semanticdb and the Scala presentation compiler.
 
-## Project structure
-- `language-server` contains a Scala implementation of a language server
-- `vscode-extension` contains a Visual Studio Code extension, implementing a client for the language server
 
-## Features
+:warning: This project is very alpha stage.
+Expect bugs and surprising behavior.
+Ticket reports and patches are welcome!
 
-- [x] Scalafix lint on compile (requires `scalac-semanticdb` compiler plugin, see [instructions here](http://scalameta.org/tutorial/#semanticdb-scalac))
-- [ ] Document formatting with scalafmt
 
-... WIP
+## Roadmap
 
-## Trying this out
-- publish the `language-server` locally
+Below is a rough summary of what features work and don't work.
+Some of those features are a likely outside the scope of this project, we are still learning and exploring what's possible.
+Please share your thoughts in
+[#2](https://github.com/scalameta/language-server/issues/2).
 
-```sh
-cd language-server
-sbt publishLocal
-```
+- [x] Linting with Scalafix
+- [x] Formatting with Scalafmt
+- [x] Auto completions as you type with presentation compiler
+- [x] Go to definition from project Scala sources to project Scala sources with Semanticdb
+- [ ] Go to definition from project sources to dependency sources
+- [ ] Go to definition from dependency sources to dependency sources
+- [ ] Go to definition in Java sources
+- [ ] Show red squigglies as you type
+- [ ] Show red squigglies on compile
+- [ ] Show parameter list as you type, signature helper
+- [ ] Show type at position
+- [ ] Find symbol references
+- [ ] Show docstring on hover
 
-- open the vscode-extension in Visual Studio Code
+## Contributing
 
-```sh
-cd vscode-extension
-code .
-```
+See the [contributing guide](CONTRIBUTING.md).
 
-and press F5 to start debugging.
+### Team
+The current maintainers, people actively working on the project and can merge pull requests, are:
```

Comment:
I think there is a typo in this sentence, probably some word missing?

Reply:
It was bad wording, good catch!